interface/src/app/analyze/analysis/analysis.js

```diff
@@ -202,7 +202,7 @@ function AnalysisCtrl($scope, $log, $q, $state, SampleBin) {
         'properties': {
           'enter': {
             // TODO dynamic layout: compute these constants also (see above)
-            'text': {'value': 'node'},
+            'text': {'value': 'signature'},
```

Comment:
I made these changes and I can see the heatmap now! But when I click "Cluster Signatures" button, I got this error:
```
TypeError: Cannot read property 'value' of undefined
```
I suspect this is due to some similar replacement. Can you take a look?